### ISSUE TITLE
Unable to move narrator cursor in scan mode for pre-populated TextField.

### ISSUE THREAD

**Issue by SarveshNamjoshi** [microsoft/fluentui#12676]
**Environment Information**
         Package version(s): current
         Browser and OS versions: Microsoft Edge Version 81.0.416.34 (Official build) beta (64-bit)

**Please provide a reproduction of the bug in a codepen:**
Repro-steps : 1. Open the Office UI Fabric Home site (TextField) in Microsoft Edge (Chromium) browser: https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield
              2. In **Standard** TextField, put value "123" (find screenshot attached), and click outside in order to focus out of the TextField. 
			  3. Turn Windows narrator on (with scan mode 'on')
			  4. Use "Down" button to navigate till Standard TextField label (narrator will read "Standard")
			  4. Use "Down" button to navigate till Standard TextField edit box (narrator will read "Unable to move narrator cursor").
			  
**Actual behavior:**
For the pre-populated TextField, unable to move narrator cursor in scan mode.

**Expected behavior:**
For the pre-populated TextField, narrator cursor should move in scan mode.

**Priorities and help requested:**
Requested priority: High

[TextField_Screenshot.docx](https://github.com/microsoft/fluentui/files/4473810/TextField_Screenshot.docx)

**Comment by joschect** [comment]
@SarveshNamjoshi Thanks for submitting this issue! I can reproduce this. We'll get this added to our backlog and get it taken care of. 

This is an interesting bug, Narrator should be dropping out of scan mode when focus goes to an input field, but it doesn't seem to do that.

**Comment by SrikanthBoyapati** [comment]
Do we have an update on this please? Any idea on when this issue might be fixed?

**Comment by xugao** [comment]
@smhigley - have you seen this issue before / know what might be causing this? (I thought it's an issue with Narrator but I couldn't repro on a plain html input)

**Comment by smhigley** [comment]
Well that's weird 😅. I don't think it's the text input itself, something else is going on on that page. I can't repro with a copy/paste of the form field HTML, and Narrator keeps freezing when I try to go in an out of the Usage example region. It also can't be some random JS thing, since it still repros on that page with JS disabled.

I'll poke at it a bit more later. @SarveshNamjoshi did this come up only on the Fluent example page, or was this reproing in another environment as well?

**Comment by xugao** [comment]
@smhigley - i think this is a Narrator bug. i simplified the repro a bit [here](https://codepen.io/xugao/pen/yLeZevr). It crashes the browser for me. I will file a bug to their team

**Comment by xugao** [comment]
closing this as it's confirmed to be a Narrator bug [here](https://microsoft.visualstudio.com/OS/_workitems/edit/27838679)
